portfolio/src/main/webapp/smwapinza sps Spring21-90676b17e7a6.json

```diff
@@ -0,0 +1,12 @@
+{
+  "type": "service_account",
+  "project_id": "smwapinza-sps-spring21",
+  "private_key_id": "90676b17e7a62ce5d2cc062c8524e2e2e22509f4",
+  "private_key": "-----BEGIN PRIVATE KEY-----\nMIIEvAIBADANBgkqhkiG9w0BAQEFAASCBKYwggSiAgEAAoIBAQDGRndrmENS0SAS\n3IxCsyO56LvSingcQQ1+3nNjsQc19QLgukOHVnSq5MJ2OIq/W9aRwzFZ1VyP881o\nV+9PJQJe6YXQVrf58dsYM+kCXYylPW6mtM5uEz6flzAR5UuyVGwGours7SXzSnxI\nbCwSE9dA7w+d3HkqtCd1xLwMs9zMd/p3IBwZJ6FTxW1gel9kzv6Wsd790ZRjTjZ+\nKbgf2QbEdj3E9vErDv58dv9Q1b9op9I1YukjxpWjmVaw6cTMgylKrAZ5bAKpnhRK\nbdM+LvaXJjlRlBCvTgrl/UYyB3Ld0pHWdHjIsAwrhIQbC656HLFlbkgBgR8Zlnru\np2cAXXVlAgMBAAECggEABA/EeX1Ef8lN5WUEMIeX50KHum/gtv90JdGrvnKNacML\nnIvkPzz7pdaKzz4oRXQ9qH29oI/41fASGJ2YhuAEtOoGtjZYjXOxX8UMdGJzM51X\nC24vvlyBj2Za8zrlupD1J4ul0VWoTdwMQLWKj5I9ehSLltnB16iaGFT/3yi+EKgp\nSBY5Pl4V3kxtZO8QSUG026zpVKMstIZshhF3Bk2c+FXwBZ7ROF+qzNf8K49U/LHr\n1x0JDgSPXbZN5o9tnP7eyJy270Ye2FhvyCME+FzaGofXkbnPsekwr3Flkv+d4BOt\nwtVEwM5hI/PpgyfGJDkfq9LV4rxgPl/okap38Sgv3wKBgQDuXIIdSN6ttcBz6+mP\nDiXBAR0mjUCvN613ZloUdP1At0GnwbvDUOxaq3vBa3LxpXDIg7A6c/9lHSDD4g2Z\nNE1+bCE9nQ1RF7Z54saV2Wiqbxuxl6BzHTPDDnDTKhL2Sf18VjwhRhk2CDhvZgZm\n8HRCuHDZecW05Aheb00LG/nvYwKBgQDU8pJM4iWhEjAg9FXZVc3tItSBOdZWBS3x\nPueNl3ovIYgKIZTVVgJhW/B3GBvbxebxFrEHiZ5PFDc9sPRazfyxi/lgFFOcaPK7\n88CUwG9BHmv0Blho5e6wmaeGHrS5qAj7eDSbYZMg0xo47NaBB/BXnMe2gG+esjya\nPiFShkJWlwKBgFv3aWRvJyuSwqx3S2gcp0hua5biTSDNtKoq3XqjIMuFaVmWkCFt\n6aOaImegU3sUwA09chM7ikCzGl85qtF/sp6ijbi+DdNJ+OfSLZjHh3Cq6d59iDSi\nAcSyn6l5xFMufRUqGI0mBLgqN8wTa0jwYYsUqsXeDAAdM3VsMCenJanNAoGAc+bL\nBhmYTYCuvWD/UNlIyabJbsRiClOrW8DXGtO1Zu2jM5Bvh2XWydLWoJt4njeqqNGm\n7OlURspD/snPVZUD0kbTJh4qG9lrNC1JRi0u4IyEST8i44UyUuBus6sGxn3U1bnp\nOCD1iUoc3ZrabUQ65hSpBoYFOqTcIz9pt5bniGUCgYBsSmxaOjt4dGNxcTq9PXhf\nrSFDXCJDDsmBR3n9VO3TinY2z+Znn5+6fv1PVISE+3ojaIiisqR7+8Y/BwxR6CVq\nBUG2RoqnFkHGs2L/PElhbmOZ+kIh0SXxHAClDlGToLiPdf0yIVkgBtYtx9v5+ukW\nMdvl4QxsyWKJr1aYFrri+Q==\n-----END PRIVATE KEY-----\n",
```

Comment:
You might have noticed you got an email about this key being leaked online. The reason is these are private keys and anyone who has this can pose as your cloud project and start serving API endpoints and doing evil things like that. 

To fix this, you'll first want to delete your key from the cloud console and generate a new one. Since this is already public, a bad actor can use it and get free usage.. Then delete this file in your repo and remove it from github. You can do that by putting this file into your .gitignore, which basically tells git to not include these files when committing. Therefore, you'll still have the file locally (you'll need it for things to work..) but they won't be pushed to github so other people can't see them.

No worries, the instructions are pretty unclear about this, but you can find instructions by looking up "cloud keys uploaded to GitHub" in the Technical Project Chat. that thread should help